Makefile

```diff
@@ -28,7 +28,7 @@ build-workspace-optimizer-x86_64: use-rust-optimizer-multi
 	docker buildx build --platform linux/amd64 -t $(DOCKER_NAME_WORKSPACE_OPTIMIZER):$(DOCKER_TAG) --target workspace-optimizer --load .
 
 build-workspace-optimizer-arm64: use-rust-optimizer-multi
-	docker buildx build --platform linux/arm64 -t $(DOCKER_NAME_WORKSPACE_OPTIMIZER):$(DOCKER_TAG) --target workspace-optimizer --load .
+	docker buildx build --platform linux/arm64/v8 -t $(DOCKER_NAME_WORKSPACE_OPTIMIZER):$(DOCKER_TAG) --target workspace-optimizer --load .
```

Comment:
👍🏼  I hope this was compatible / equivalent.

Reply:
Alright, I wait for your confirmation then.

I changed this for consistency.

Reply:
It seems `linux/arm64/v8` is (currently) just a synonym of `linux/arm64`.

Reply:
okay, great. Then we have the same everywhere now.